src/js/@WebParts/ProgramProjectsTimelineSync/index.tsx

```diff
@@ -349,24 +349,43 @@ export default class ProgramProjectsTimelineSync extends React.Component<IProgra
         return valuesToSync;
     }
 
+    /**
+     * Get web information of project from source
+     *
+     * @param {common.ProjectItem} project Project
+     */
+    private async getProjectWebInformation(project: common.ProjectItem) {
+        try {
+            const projectWeb = new Web(project.URL);
+
+            const projectWebTitle = await projectWeb.get();
+
+            return projectWebTitle;
```

Comment:
You're returning projectWebTitle, so for someone else looking at this it would seem this is the actual project web title as a string. Should returning the actual title with this variable name.